### ISSUE TITLE
Unable to install spotweb docker on ARM

### ISSUE THREAD

**Issue by Alootje** [edv/docker-spotweb#41]
For some time I've been trying to install the Spotweb docker on my ARM system ([Rockpi 4c](https://wiki.radxa.com/Rockpi4)). I've tried multiple methods but keep running into issues.
The system is running ARMbian and has Openmediavault, Docker and Portainer installed.

**First I tried setting up a database (linuxserver/mariadb) and used the Quick setup using dockerfile approach**
`docker run -p 8080:80 --name spotweb -d -v /etc/localtime:/etc/localtime:ro erikdevries/spotweb`

This results in the follow error:
`standard_init_linux.go:211: exec user process caused "exec format error"
`

**Second I tried In Portainer.** I added the content of the [docker-compose-arm.yml](https://github.com/edv/docker-spotweb/blob/master/docker-compose-arm.yml) file as a stack. This results in the following error:
```
Deployment error
Status: no matching manifest for linux/arm64/v8 in the manifest list entries, Code: 1
```
**Third attempt is via ssh.** Now I downloaded the [docker-compose-arm.yml] onto my system and ran `docker-compose -f docker-compose-arm.yml up`. Again, resulting in an error:
`ERROR: yaml.scanner.ScannerError: mapping values are not allowed here`

So basically the hair which not yet has fallen out of my head is turning grey. Can someone give me a push in the right direction?

Three side questions, (if desired I can make separate issues):
1. in the docker-compose.yml I see that for the database hypriot/rpi-mysql:5.5 is used. This container has not been updated in two years. Isn;t it better to update this to mysql/mysql-server:8.0.21-1.1.17? 
2. Can [MariaDB](https://hub.docker.com/_/mariadb) be used Instead of MySQL?
3. How can I access the spotweb website from outside my LAN with a reverse proxy?

**Comment by Alootje** [comment]
Nobody any idea?

**Comment by edv** [comment]
From the example it looks like you try to run the x86 docker container and not the arm version. Use the erikdevries/rpi-spotweb container and let me know if that does work.

**Comment by dennismark** [comment]
I had the same problem and indeed I used the x86 docker container. Thank you for the tip
